app/users/reducer.js

```diff
@@ -0,0 +1,92 @@
+import {
+  TOGGLE_LOGIN_MODE,
+  MANUAL_LOGIN_USER,
+  LOGIN_SUCCESS_USER,
+  LOGIN_ERROR_USER,
+  SIGNUP_USER,
+  SIGNUP_SUCCESS_USER,
+  SIGNUP_ERROR_USER,
+  LOGOUT_USER,
+  LOGOUT_SUCCESS_USER,
+  LOGOUT_ERROR_USER } from './actions';
```

Comment:
Are moving towards explicitly importing the required actions?

I like the idea of exporting constants within the `actions.js` file, but that can quickly become a large file. Perhaps we could still keep `types` within each module.

```javascript
component/
     -reducers
     - types
     - actions
     - index
     - components/
         - Container
```